GitUI/CommandsDialogs/GitIgnoreDialog/GitIgnoreModel.cs

```diff
@@ -22,15 +22,12 @@ public class GitIgnoreModel : Translate, IGitIgnoreDialogModel
         private readonly TranslationString _saveFileQuestion =
             new TranslationString("Save changes to .gitignore?");
 
-        private readonly IGitModule _module;
         private readonly IFullPathResolver _fullPathResolver;
 
         public GitIgnoreModel(IGitModule module)
```

Comment:
💭 Seems like this parameter could be simplified to just a working directory, but it's also possible that using `IGitModule` make it more difficult to misuse.

Reply:
we shouldn't be injecting `IGitModule` as is - it is a mixture of state and behavior.
In other places it is done as `Func<IGitModule>` to remove/reduce a risk of executing an operation against a stale instance